### PR TITLE
Add brainvol stats to the segstats command

### PR DESCRIPTION
**Pull request by dkuegler**
This PR adds all the brainvol stats measures to segstats.py. This specifically includes eTIV and summary volumes.

It also adds `mri_segstats.py` and `mri_brainvol_stats.py`, which mimic freesurfer binaries, based on this code. They are much faster and use similar command line APIs (as compared to segstats.py, which has a different API).

### CODE REVIEW

File: run_fastsurfer.sh [hunk 1]
```diff
@@ -267,246 +273,112 @@ do
...
+  --threads) threads="$1" ; shift ;;
+  --py) python="$1" ; shift ;;
+  -h|--help)
     usage
     exit
     ;;
-    --version)
-    if [[ "$#" -lt 2 ]]; then
+  --version)
+    if [[ "$#" -lt 2 ]] || [[ "$1" =~ ^-- ]]; then
```
Comment (dkuegler) [inline run_fastsurfer.sh]:
```suggestion
    if [[ "$#" -lt 1 ]] || [[ "$1" =~ ^-- ]]; then
```

File: CerebNet/inference.py [hunk 2]
```diff
@@ -291,6 +291,7 @@ def _get_ids_startswith(_label_map: Dict[int, str], prefix: str) -> List[int]:
         table = pv_calc(
             seg_data,
             norm_data,
+            norm_data,
```
Comment (m-reuter) [inline CerebNet/inference.py:294]: why do we have norm_date duplicated here?

Comment (dkuegler) [reply CerebNet/inference.py:294]: These are the values for `norm_data` and `pv_guide`. One is used to calculate the intensity values the other to determine the pv corrected volume.

Mri_segstats uses -pv and -norm as equivalent flags.

Comment (dkuegler) [reply CerebNet/inference.py:294]: ... And in CerebNet (here) this is actually the same data (usually also in FreeSurfer, but this is not required)

**Pull request merged**